src/librustc_typeck/diagnostics.rs

```diff
@@ -364,6 +364,76 @@ it has been disabled for now.
 [iss20126]: https://github.com/rust-lang/rust/issues/20126
 "##,
 
+E0197: r##"
+Inherent implementations (one that do not implement a trait but provide
+methods associated with a type) are always safe because they are not
+implementing an unsafe trait. Removing the unsafe keyword from the inherent
```

Comment:
Could you put `'s around the second unsafe?

    Removing the `unsafe` keyword ...